gdnative-bindings/build.rs

```diff
@@ -15,7 +15,8 @@ fn main() {
     let icalls_rs = out_path.join("icalls.rs");
 
     let api = Api::new(&api_data);
-    let binding_res = generate_bindings(&api);
+    let docs = GodotXMLDocs::new("xmls");
```

Comment:
Might be better to name the subdirectory `docs` so that its purpose is clear?

Reply:
done